modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -171,6 +171,12 @@
   "manageDocType": "Manage {{ type }}",
   "manageDraftSubmissions": "Manage Draft Submissions",
   "managePages": "Manage Pages",
+  "mediaCreatedDateDayjsFormat": "MMM Do, YYYY",
```

Comment:
There is an established pattern for localized dayjs formats. Should be something like `dayjsMediaCreatedFormat`